lib/bridge.js

```diff
@@ -0,0 +1,649 @@
+const fs = require('fs');
+const util = require('util');
+const EventEmitter = require('events');
+
+const Q = require('q');
+const _ = require('busyman');
+const Zive = require('zive');
+const zclId = require('zigbee-bridge-definitions');
+const proving = require('proving');
+const Objectbox = require('objectbox');
+const debug = {
+    shepherd: require('debug')('zigbee-shepherd'),
```

Comment:
I think now we can change `zigbee-shepherd` to `zigbee-bridge` in that place

Reply:
done